test/test_mag_pruning.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+
+from litgpt.model import GPT, Config
+from whittle.baselines.pruner import NMPruner
+
+
+def test_pythia():
```

Comment:
Could we have only one function that loops over the different models? See https://github.com/whittle-org/whittle/blob/6367d76c7053372d2b6458300be5c540e1f2f1c6/test/test_training_strategies.py#L82